### PR TITLE
fix(recipes-react-components): Add a FluentProvider to the local storybook

### PR DESCRIPTION
**Pull request by sopranopillow**
## Previous Behavior

When rendering a recipe in the local storybook there was no FluentProvider, therefore the Fluent components weren't styled correctly.

## New Behavior

Recipes are wrapped with a FluentProvider in the local storybook. Note that the provider is added by the local storybook, outside of the local package the change will take no effect.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2e4a9ee0c93e252ac19771d934b5f9e233faddd7'>2e4a9ee0c93e252ac19771d934b5f9e233faddd7</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/recipes-react-components/src/DocComponents/FluentDocsContainer.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { FluentStoryContext } from '@fluentui/react-storybook-addon';
+import { webLightTheme, FluentProvider } from '@fluentui/react-components';
+
+type FluentDocsContainerProps = {
+  context: FluentStoryContext & DocsContextProps;
+};
+
+export const FluentDocsContainer: React.FC<FluentDocsContainerProps> = ({ children, context }) => {
+  return (
+    <FluentProvider style={{ backgroundColor: 'transparent' }} theme={webLightTheme}>
```
Comment (spmonahan) [inline apps/recipes-react-components/src/DocComponents/FluentDocsContainer.stories.tsx:12]: Why is the transparent background necessary here?

Comment (Hotell) [reply apps/recipes-react-components/src/DocComponents/FluentDocsContainer.stories.tsx:12]: this looks like copy paste of `apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx` can you please create a feature request to generalize these sb utils under one package to have DRY codebase ?

Comment (sopranopillow) [reply apps/recipes-react-components/src/DocComponents/FluentDocsContainer.stories.tsx]: Also removed the transparent style since it wasn't needed.

**Review by Hotell — APPROVED** [review]

**Pull request merged**